bigquery/google/cloud/bigquery/client.py

```diff
@@ -1355,6 +1355,11 @@ def load_table_from_uri(
 
         Returns:
             google.cloud.bigquery.job.LoadJob: A new load job.
+
+        Raises:
+            ValueError:
+                If ``job_config`` is not an instance of ``google.cloud.bigquery.job.LoadJobConfig``
```

Comment:
IMO it would be convenient to cross-link the expected class in the docs (applies to multiple places):

```suggestion
                If ``job_config`` is not an instance of :class:`~google.cloud.bigquery.job.LoadJobConfig`
```